[{"name": "test.test_summary.Summary.test_calls", "status": "passed", "message": "", "passed": true, "points": ["p02-05.1"], "backtrace": []}, {"name": "test.test_summary.Summary.test_main", "status": "passed", "message": "", "passed": true, "points": ["p02-05.1"], "backtrace": []}, {"name": "test.test_summary.Summary.test_missing_file", "status": "passed", "message": "", "passed": true, "points": ["p02-05.1"], "backtrace": []}, {"name": "test.test_summary.Summary.test_one", "status": "failed", "message": "7.964521328993977 != 8.904606 within 4 places (0.940084671006022 difference) : Wrong std for file example.txt!", "passed": false, "points": ["p02-05.1"], "backtrace": ["  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 57, in testPartExecutor\n    yield\n", "  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 623, in run\n    self._callTestMethod(testMethod)\n", "  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 579, in _callTestMethod\n    if method() is not None:\n       ^^^^^^^^\n", "  File \"C:\\Program Files\\Python Project\\P1\\PythonData-1\\øvelser\\e25_summary\\test\\test_summary.py\", line 25, in test_one\n    self.assertAlmostEqual(d, 8.904606, places=4, msg=\"Wrong std for file example.txt!\")\n"]}, {"name": "test.test_summary.Summary.test_three", "status": "failed", "message": "40.8248290463863 != 50.0 within 4 places (9.175170953613701 difference) : Wrong std for file example3.txt!", "passed": false, "points": ["p02-05.2"], "backtrace": ["  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 57, in testPartExecutor\n    yield\n", "  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 623, in run\n    self._callTestMethod(testMethod)\n", "  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 579, in _callTestMethod\n    if method() is not None:\n       ^^^^^^^^\n", "  File \"C:\\Program Files\\Python Project\\P1\\PythonData-1\\øvelser\\e25_summary\\test\\test_summary.py\", line 39, in test_three\n    self.assertAlmostEqual(d, 50.000000, places=4, msg=\"Wrong std for file example3.txt!\")\n"]}, {"name": "test.test_summary.Summary.test_two", "status": "failed", "message": "2550.9754056046877 != 3124.294045 within 4 places (573.3186393953124 difference) : Wrong std for file example2.txt!", "passed": false, "points": ["p02-05.1"], "backtrace": ["  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 57, in testPartExecutor\n    yield\n", "  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 623, in run\n    self._callTestMethod(testMethod)\n", "  File \"C:\\Program Files\\Python311\\Lib\\unittest\\case.py\", line 579, in _callTestMethod\n    if method() is not None:\n       ^^^^^^^^\n", "  File \"C:\\Program Files\\Python Project\\P1\\PythonData-1\\øvelser\\e25_summary\\test\\test_summary.py\", line 32, in test_two\n    self.assertAlmostEqual(d, 3124.294045, places=4, msg=\"Wrong std for file example2.txt!\")\n"]}]